src/components/Sidebar/PostDetails/PostDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+import { NewCommentForm } from './NewCommentForm';
+import {
+  ActivePostContext,
+  CommentFormContext,
+  CommentListContext,
+} from '../../../utils/Store';
+import { CommentList } from './../CommentList/CommentList';
+
+export const PostDetails: React.FC = () => {
+  const { activePost } = useContext(ActivePostContext);
+  const { comments } = useContext(CommentListContext);
+  const { isActiveForm } = useContext(CommentFormContext);
+
+  return (
+    <div className="content" data-cy="PostDetails">
+      <div className="block">
+        <h2 data-cy="PostTitle">{`#${activePost?.id}: ${activePost?.title}`}</h2>
+
+        <p data-cy="PostBody">{activePost?.body}</p>
+      </div>
+
+      {activePost && <CommentList comments={comments} />}
+
+      {isActiveForm && <NewCommentForm />}
```

Comment:
The component `NewCommentForm` should have its propTypes defined to ensure that the expected props are passed correctly. Please make sure to describe the props for `NewCommentForm` using propTypes.